### PR TITLE
Announce Hammer-8 release

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@cybette smoke testing has completed successfully.
Can you review/merge as time permits.

File: site/_posts/2019-07-02-Announcing-Hammer-8.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+We've just built Hammer-8. This release contains bug fixes, UI tweaks, and stabilization.
+Here are just a few of the things added since Hammer-7 release:
+
+- ADDED
+  * Add migration analytics reports
+
+- FIXED 
+  * Group AND expressions properly to account for nesting
+  * [V2V] Modify active_tasks so that it always reloads
+  * EvmDatabaseOps] Fix .validate_free_space target
```
Comment (cybette) [inline site/_posts/2019-07-02-Announcing-Hammer-8.md]: minor thing, but I think this is missing an opening square bracket `[`

Comment (JPrause) [reply site/_posts/2019-07-02-Announcing-Hammer-8.md]: Good eyes. :eyes:

Comment (cybette) [reply site/_posts/2019-07-02-Announcing-Hammer-8.md]: It's a blessing and curse. Anyway thanks for the post, I will merge once the checks have completed!

**Pull request merged**